BREAKING.md

```diff
@@ -86,9 +72,6 @@ The following packages use React and thus were impacted:
 
 Users of these packages may need to update to React 17, and/or take other action to ensure compatibility.
 
-### Remove `documentId` field from `MockFluidDataStoreContext`
-This field has been deprecated and will be removed in a future breaking change.
-
 ### IntervalCollection event semantics changed
 
 The semantics of events emitted by IntervalCollection were changed to be more consistent:
```

Comment:
Does "changed" imply breaking change? The statement here should be around "breaking" element of the change and what user needs to do to accommodate for the change.

Reply:
Pasting the linked note:
```md
### bindToContext to be removed from IFluidDataStoreChannel
`bindToContext` will be removed from `IFluidDataStoreChannel` in the next major release.
It was deprecated in 0.50 but due to [this bug](https://github.com/microsoft/FluidFramework/issues/9127) it still had to be called after creating a non-root data store. The bug was fixed in 0.59.
To prepare for the removal in the following release, calls to `bindToContext` can and should be removed as soon as this version is consumed. Since the compatibility window between container runtime and data store runtime is N / N-1, all runtime code will have the required bug fix (released in the previous version 0.59) and it can be safely removed.
```

Looks like there is no replacement for the removed item?